code/_onclick/item_attack.dm

```diff
@@ -68,6 +68,11 @@
 		return TRUE
 	interact(user)
 
+/obj/item/proc/attack_self_secondary(mob/user, modifiers)
+	if(SEND_SIGNAL(src, COMSIG_ITEM_ATTACK_SELF_SECONDARY, user) & COMPONENT_CANCEL_ATTACK_CHAIN)
+		return TRUE
+	interact(user)
```

Comment:
Remove this interact, right-click being the same as left-click is something I want to avoid going forward, and I doubt anyone's right-clicking on items now.